galeracluster/source/mysqlwsrepoptions.rst

```diff
@@ -1010,7 +1017,11 @@ Defines whether the node participates in replication.
 | **Support**             | *Introduced:*       | 1                                 |
 +-------------------------+---------------------+-----------------------------------+
 
-This parameter defines whether or not the node participates in replication.  When set to ``OFF``, the node functions as a standard standalone MySQL database server.  No transactions made during this session replicate to the cluster.
+This parameter defines whether or not the node participates in replication.  When set to ``OFF``, the node remains in the cluster but ceases replication, functioning as a standard standalone MySQL database server.  No transactions made during this session replicate to the cluster.
```

Comment:
This parameter defines whether or not updates made in the current session are replicated. "When set to OFF ... " etc can be removed.